### ISSUE TITLE
Chromebook installation

### ISSUE THREAD

**Issue by robvh7** [ZaneYork/SMAPI-Android-Installer#106]
Hi, is it at all possible to download the newly created files for smapi manually so we can add them/copy them to the stardew valley folder?? Just like you can do with smapi on windows.. Many thanks

**Comment by GeekieFreak** [comment]
No I don't think so. Are you trying to install SMAPI on Chromebook?

**Comment by GeekieFreak** [comment]
You could also just drag them to the folder...

**Comment by robvh7** [comment]
Yes trying to install on Chromebook. I can install apks via the Linux beta terminal and the smapi installer install fine. It's just when I open the installer to install the smapi bits that it says need to be in developer mode for that function
